core/modules/update/update.module

```diff
@@ -445,7 +445,8 @@ function _update_get_cached_available_releases() {
 function update_mail($key, &$message, $params) {
   $language = $message['language'];
   $langcode = $language->langcode;
-  $message['subject'] .= t('New release(s) available for !site_name', array('!site_name' => config_get_translated('system.core', 'site_name'), array(), array('langcode' => $langcode)), array('langcode' => $langcode));
+  $site_name_localized =  config_get_translated('system.core', 'site_name', array(), array('langcode' => $langcode));
```

Comment:
Extra space here after '=' can be removed.

Reply:
Oops, correct, I'll remove that.